packages/components/src/styles/components/code-editor/theme.scss

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+// THEMING
+
+.hds-code-editor--theme-dark {
+  // COLORS
+
+  // Base UI colors:
+  --hds-code-editor-color-border-strong: rgba(178, 182, 189, 40%);
+  --hds-code-editor-color-foreground-primary: #d5d7db;
+  --hds-code-editor-color-foreground-faint: #878a8f;
+  --hds-code-editor-color-foreground-high-contrast: #fff;
+  --hds-code-editor-color-surface-faint: #15181e;
+  --hds-code-editor-color-surface-primary: #0D0E12;
```

Comment:
Is there a reason why these colors are RGBA/HEX values, vs tokens (local) that are shared among this component and the CodeBlock? Both of them are visually the same, and wouldn't creating shared tokens make maintenance easier?

Reply:
I'm not sure what you mean by local tokens. This is also how we set up the colors for the CodeBlock. Do you have something you could point me to?